people/Proselyte093/page.en.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Proselyte
+---
+
+## Description
+
+Proselyte is our ally. They often interact with us friendly.
+
+They are fond of cute cats. Whether it's their own cat, or cats in a pet shop, or cats they meets on the street.
+
+They also liked some niche pixel art RPG games, such as *Ib* and *Yume 2kki*. At the same time, they followed *Yume Nikki Online Project* on Github.
+
+## Departure
+
+Owing to their family and studies, Proselyte was troubled by anxiety and depression all along. They felt that they were losing the sense of exist.
+
+However, they still supported theirselves by viewing good things in a long term, incuding their favourite games, cute animals, and colourful illustrations. When our friends fell in anxiety, they would generously give their warm hug. 
+
+Until early November, they still thought that curing depression before their birthday. Unfortunally, their mental symptoms gradully overflowed. In an evening, they went out after taking medicine, then departed due to hypothermia at that night.
```

Comment:
Until early November, they still held on to the hope of curing depression before their birthday. Unfortunally, their mental health gradully worsened. In an evening, they went out of their home and assimilated into the night.